tests/helper/skyMapTestCase.py

```diff
@@ -30,6 +30,30 @@
 from lsst.skymap import skyMapRegistry
 
 
+def checkDm14809(testcase, skymap):
+    """Test that DM-14809 has been fixed
+
+    The observed behaviour was:
+
+        skyMap.findTract(skyMap[9712].getCtrCoord()).getId() != 9712
+
+    and
+
+        skyMap[1].getCtrCoord() == skyMap[11].getCtrCoord()
+
+    In order to be thorough, we generalise these over the entire skymap.
+    """
+    # Check that the tract ID is that used to index the skymap
+    expect = [tract.getId() for tract in skymap]
+    got = [skymap.findTract(tract.getCtrCoord()).getId() for tract in skymap]
+    testcase.assertListEqual(got, expect)
```

Comment:
I think this test does what the comment below says. I am not sure it does what its own comment says, I would have expected a test for the comment above to look like this:
```
expect = range(len(skmap))
got = [tract.getId() for tract in skymap]
testcase.assertEqual(expect, got)
```
Also I recommend using `assertEqual` instead of `assertListEqual`; it works fine on lists and is the technique recommended by the docs for the `unittest` module (though I believe it does delegate to `assertListEqual` for lists).